app/models/news.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 
 class News < ApplicationRecord
+    validates :title, uniqueness:true
```

Comment:
didn't we discuss that it's better to check for uniqueness of a link?

Reply:
oh yes, i did it